packages/extension-api/src/extension-api.d.ts

```diff
@@ -423,6 +423,10 @@ declare module '@podman-desktop/api' {
     export const onDidUnregisterContainerConnection: Event<UnregisterContainerConnectionEvent>;
     export const onDidRegisterContainerConnection: Event<RegisterContainerConnectionEvent>;
     export function getContainerConnections(): ProviderContainerConnection[];
+    export function getMatchingProviderLifecycleContext(
```

Comment:
```suggestion
    export function getProviderLifecycleContext(
```

probably matching is too internal 
WDYT ?

Reply:
in the jsdoc it does not say if there is an exception if there is no context ?

do we get an error or we should return undefined in that case ?

Reply:
What do you mean by jsdoc? Adding a comment to the signature?

Currently it throws an error when there is no context.

Reply:
yes comments 👍